store/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,108 @@
+databaseChangeLog:
+  - changeSet:
+      id: createBookTable
+      author: your_author_name
```

Comment:
see how it should looks like
![image](https://github.com/artificialLeo/mate-bs/assets/20326858/89fda2da-e034-432d-b23e-3ca68dd54b99)